lib/admin_user_index_query.rb

```diff
@@ -69,6 +69,8 @@ def initialize_query_with_order(klass)
       query = query.includes(:user_stat)
     end
 
+    query = query.joins(:primary_email) if params[:show_emails]
```

Comment:
We should probably check that this value is `true`? I know recently we fixed a case with a different param where they passed in `false` and it still worked.